app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  validates :email, presence: true, uniqueness: true
```

Comment:
Have we tested that this uniqueness constraint works as intended?

Reply:
It works, but I don't have a test yet for this